deployments/pod-mlnx-dpdk.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mlnx-dpdk
+  annotations:
+    k8s.v1.cni.cncf.io/networks: sriov-net1, sriov-net1
```

Comment:
let do a simple that we add 1 network, so replace "k8s.v1.cni.cncf.io/networks: sriov-net1, sriov-net1"
with k8s.v1.cni.cncf.io/networks: sriov-net1

Reply:
fixed